public/index.html

```diff
@@ -0,0 +1,370 @@
+                </li>
+                <li class="flex-item-4">
+                    <h2>JESSIE DOE</h2>
+                    <h3 class="team-title">Bussines and legacy</h3>
+                    <img class="face-pic" src="./imgs/jessiedoe.png" alt="head picture of Jessie Doe">
+                    <ul class="social-media">
+                        <li><a href=""><span class="socicon">b</span></a></li>
+                        <li><a href=""><span class="socicon">a</span></a></li>
+                        <li><a href=""><span class="socicon">D</span></a></li>
+                        <li><a href=""><span class="socicon">j</span></a></li>
+                        <li><a href=""><span class="socicon">c</span></a></li>
+                    </ul>
+                    <p class="team-para">Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+                </li>
+        </ul>
+        <h2 class="subheader">CLIENTS</h2>
+        <div class="client-wrapper">
+            <a href=""><img class="arrow" src="./imgs/l-arrow.png"></a>
+            <div class="client-section">
+                <img class="client-pic" src="./imgs/mattbroderic.png" alt="">
+                <div class="client-text-wrapper">
+                    <h3><span class="client-name">Matthew Broderic </span><span class="client-title">(CEO Hopicola Inc.)</span></h3>
+                    <p class="client-text">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+                </div>
+            </div>
+            <a href=""><img class="arrow" src="./imgs/r-arrow.png"></a>
+        </div>
+    </section>
+    <section class="section-orange contact-us">
+        <object class="clip clip-white" type="image/svg+xml" data="./imgs/clipwhite.svg">
+        </object>
+        <a id="contact"></a>
+        <h1>CONTACT</h1>
+        <p>Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+        <div class="flex-container map-container">
+            <form class="flex-item-1">
+                <input type="text" placeholder="Company name" id="form_companyname">
+                <input type="email" placeholder="Your@mail.com" id="form_email">
+                <textarea name="message" id="contact-form" placeholder="Message..."></textarea>
+                <button class="message-button" type="submit">SEND MESSAGE</button>
+            </form>
+            <div class="map flex-item-1">
+                <div></div>
+            </div>
```

Comment:
Where's the map content? Why is all the important contact information in an image, which is only visible on rollover? Why is it cutoff on mobile? How are mobile customers supposed to contact us, let alone customers using assistive technology?